src/main/java/uk/gov/hmcts/opal/service/proxy/ProxyInterface.java

```diff
@@ -5,7 +5,7 @@
 /** A marker interface to help identify code that is to be removed at a future date. */
 public interface ProxyInterface {
 
-    public static final String LEGACY_MODE = "LEGACY";
+    String LEGACY_MODE = "LEGACY";
```

Comment:
Why change this from static final? As a static final, it allows the compiler to optimise.

Reply:
Interface variables are static and final by default in Java.